cmd/aspect/clean/clean.go

```diff
@@ -53,16 +51,25 @@ Workaround inconistent state:
 	Such problems are fixable and these bugs are a high priority.
 	If you ever find an incorrect incremental build, please file a bug report,
 	and only use clean as a temporary workaround.`,
-		RunE: b.Run,
+		RunE: func(cmd *cobra.Command, args []string) (exitErr error) {
+			return pathutils.InvokeCmdInsideWorkspace("clean", func() error {
```

Comment:
Instead of hardcode `"clean"`, you can do:

```suggestion
			return pathutils.InvokeCmdInsideWorkspace(cmd.Use, func() error {
```